### PR TITLE
Trade Chart Refinement

### PR DESCRIPTION
**Pull request by HeeJu-K**
Chart functional bug and UI improvement

functional: 
added token selection tab when no pair, token selection from modal and tab can function correctly

UI: 
timescale, candle number, etc
fixed candle number display, but still only two candles are shown when selected from market

### CODE REVIEW

**Comment by HeeJu-K** [comment]
Remaining bug on futures&option:
- option page activeSymbol changes to BTC-60000-C when BTC-10000-C is selected when sti1m is called, still seeking for possible reasons
- option should show chart when wallet not connected
- other possibe bugs pending to be checked

**Comment by ykchong45** [comment]
1. after [1d31974](https://github.com/ACY-Labs/acy-dex-interface/pull/105/commits/1d31974bfc0cbcecdbf4dbcba803b8d67b97dd01), the horizontal axis is changed to user's time zone, however when first loading the page, the last candle is still at GMT+0. 
2. when toggling from A to B symbol, network requests for both A and B symbol are continuously fired. 
3. not sure if the "2-tab" price chart for the _trade_ page is implemented. If it's implemented, let me know how can I test it.

Thank you! 😊

File: src/constants/future_option_power.js [hunk 1]
```diff
@@ -12,18 +12,28 @@ const contracts = {
...
 
-    oracleManager: '0x1F9c627BD3E8ef0b7152e4a3fAbD0B3eF1676821',
-    pool: '0x5C842b8E75D4fC8C63f2F889d7f067aa72b8C65C',
-    poolImplementation: '0xc7e11af7478D0b20d0C960008D23C7aA45a9C476',
-    symbolManager: "0x9CC0b982f427b1dE01E46610d8247F4b10a81AAf",
-    symbolManagerImplementation: "0xbEe6eDABcC42Ed8c0B46b6fcA8Ec04decaDc0A09",
-    alp: "0x002d190863243E09e83245FEE736B0B463Af5a74",
-    symbolsLens: "0xE56710F67eAf87Cf998fb3bFf4B1332080942b2F",
-    reader: "0x99E4B9F0664c700F871C53630572B08D74a6f5B5",
+    // oracleManager: '0x1F9c627BD3E8ef0b7152e4a3fAbD0B3eF1676821',
```
Comment (ykchong45) [inline src/constants/future_option_power.js:19]: Looks like these contract address is outdated. Before your commit I updated your PR with latest code from `dev_trunk`, so the commented section should be the most updated addresses.

**Review by ykchong45 — CHANGES_REQUESTED** [review]
The chart works on all derivative product pages.

Merge into main dev branch can be done after finalizing the smart contract addresses issue.

**Comment by ykchong45** [comment]
Commit [f2e564f](https://github.com/ACY-Labs/acy-dex-interface/pull/105/commits/f2e564fd82d5679e7d5465e4a68fe42e8b3397cc) has been merged into #112 , which has been merged into `develop_perpetual` branch.

**Pull request closed without merging**